### DISCUSSION TITLE
error "getaddrinfo ENOTFOUND elasticsearch"

### DISCUSSION THREAD

**Discussion by rahulpawar1209** [telekom-security/tpotce#980] [General]
![Screenshot (113)](https://user-images.githubusercontent.com/46704318/147208977-95635b0c-dd48-4e1c-970e-44e0cb344c77.png)

can you please help me to solve ASAP?

**Comment by rahulpawar1209** [comment]
also getting pop up msg 

server.publicBaseUrl is missing and should be configured when running in a production environment. Some features may not behave correctly. See the documentation.

**Comment by t3chn0m4g3** [comment]
**ENOTFOUND** means **Elasticsearch not found** which usually is the case if there was an install error (you did check the logs as suggested when opening an issue, correct 😇 ). Likely it is a RAM issue, explained in the [Wiki](https://github.com/telekom-security/tpotce/wiki/T-Pot,-RAM-and-Elastic-Stack).

> server.publicBaseUrl

Is explained in one of the only three pinned [issues](https://github.com/telekom-security/tpotce/issues/889):
![image](https://user-images.githubusercontent.com/4318452/147234407-44001998-5a73-4d25-b9e9-615d12ade5ad.png)

**Comment by rahulpawar1209** [comment]
Resolved.
Thanks a lot, buddy 👍
